source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -298,7 +299,18 @@ - (void)didChangeVisibility:(UIButton *)button isVisible:(BOOL)isVisible
     }
     else
     {
-        button.backgroundColor = [UIColor colorWithRed:0.11 green:0.68 blue:0.97 alpha:1.0];
+        if([button isKindOfClass:[ACRButton class]])
+        {
+            ACRButton *acrButton = (ACRButton*)button;
+            if(acrButton.sentiment != ACRSentimentDefault)
+            {
+                [acrButton applySentimentStyling];
+            }
+            else
+            {
+                button.backgroundColor = [UIColor colorWithRed:0.11 green:0.68 blue:0.97 alpha:1.0];
+            }
+        }
```

Comment:
you need to reset backgroundColor even if the button is not kind of ACRButton since we are assuming that button may not be ACRButton type.